src/components/Pagination/NextPaginationItem.tsx

```diff
@@ -30,5 +40,12 @@ export const NextPaginationItem = withTheme(NextPaginationItemComponent)
 const Button = styled(ItemButton)`
   ${({ theme }: InjectedProps) => css`
     color: ${theme.palette.TEXT_GREY};
```

Comment:
[ASK] Why this color?
We set `#333`  in design.

Reply:
It's just a simple mistake.
Fixed in this commit.
https://github.com/kufu/smarthr-ui/commit/eea395a59f830512b4273d45d461914341e18755